plugins/org.jkiss.dbeaver.ui.editors.data/src/org/jkiss/dbeaver/ui/controls/resultset/view/ErrorPresentation.java

```diff
@@ -98,29 +98,6 @@ public void createPresentation(@NotNull IResultSetController controller, @NotNul
                 textWidget.setText(sqlText);
             }
         }
-
-        try {
-            boolean widthSet = false;
-            IDialogSettings viewSettings = ResultSetUtils.getViewerSettings(SETTINGS_SECTION_ERROR_PANEL);
-            String errorWidth = viewSettings.get(PROP_ERROR_WIDTH);
-            if (errorWidth != null) {
-                String[] widthStrs = errorWidth.split(":");
-                if (widthStrs.length == 2) {
-                    partDivider.setWeights(Integer.parseInt(widthStrs[0]),
-                        Integer.parseInt(widthStrs[1]));
-                }
-                widthSet = true;
-            }
-            if (!widthSet) {
-                partDivider.setWeights(700, 300);
```

Comment:
Shouldn't we left this string to set weights? Or is it set somewhere else also?

Reply:
Decided: remove saved state of weights and use default, to avoid hiding errors.

Reply:
OK